pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -102,6 +102,8 @@ EOF
       --replace 'libs = [ "sandbox" ]' 'libs = [ "/usr/lib/libsandbox.1.dylib" ]'
     '');
 
+  NIX_GCC_ARCH_FLAGS = [ ]; # it fails when compiled with -march=
```

Comment:
Could you post the error? Most likely it will effect other packages as well

Reply:
Something in skia, the CPU-optimized code and non-CPU-optimized are both included in the build and there is a naming conflict. Chromium has no error, it might be because qtwebengine uses older webkit.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/59148/commits/3a1edbfe24503d96ca97cc9ec5f683720a2b9366

Reply:
No, still failed on ```sandybridge``` (ok on ```skylake```)
```
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1955:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:92:11: error: redefinition of 'template<class T> T if_then_else(I32x8, T, T)'
 SI ATTR T if_then_else(I32 c, T t, T e)
           ^~~~~~~~~~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1911:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:92:11: note: 'template<class T> T if_then_else(I32x8, T, T)' previously declared here
 SI ATTR T if_then_else(I32 c, T t, T e)
           ^~~~~~~~~~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1955:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:502:6: error: redefinition of 'void clut(const skcms_A2B*, I32x8, I32x8, Fx8*, Fx8*, Fx8*, Fx8) [with int I = 0; int B = 8; skcms_A2B = skcms_A2B; I
32x8 = __vector(8) int; Fx8 = __vector(8) float]'
 void clut<0, 8>(const skcms_A2B* a2b, I32 ix, I32 stride, F* r, F* g, F* b, F a) {
      ^~~~~~~~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1911:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:502:6: note: 'void clut(const skcms_A2B*, I32x8, I32x8, Fx8*, Fx8*, Fx8*, Fx8) [with int I = 0; int B = 8; skcms_A2B = skcms_A2B; I32x8 = __vector(8
) int; Fx8 = __vector(8) float]' previously declared here
 void clut<0, 8>(const skcms_A2B* a2b, I32 ix, I32 stride, F* r, F* g, F* b, F a) {
      ^~~~~~~~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1955:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:514:6: error: redefinition of 'void clut(const skcms_A2B*, I32x8, I32x8, Fx8*, Fx8*, Fx8*, Fx8) [with int I = 0; int B = 16; skcms_A2B = skcms_A2B;
I32x8 = __vector(8) int; Fx8 = __vector(8) float]'
 void clut<0, 16>(const skcms_A2B* a2b, I32 ix, I32 stride, F* r, F* g, F* b, F a) {
      ^~~~~~~~~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1911:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:514:6: note: 'void clut(const skcms_A2B*, I32x8, I32x8, Fx8*, Fx8*, Fx8*, Fx8) [with int I = 0; int B = 16; skcms_A2B = skcms_A2B; I32x8 = __vector(
8) int; Fx8 = __vector(8) float]' previously declared here
 void clut<0, 16>(const skcms_A2B* a2b, I32 ix, I32 stride, F* r, F* g, F* b, F a) {
      ^~~~~~~~~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1955:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:537:6: error: redefinition of 'template<int I, int B> void clut(const skcms_A2B*, I32x8, I32x8, Fx8*, Fx8*, Fx8*, Fx8)'
 void clut(const skcms_A2B* a2b, I32 ix, I32 stride, F* r, F* g, F* b, F a) {
      ^~~~
In file included from ../../3rdparty/chromium/third_party/skia/third_party/skcms/skcms.cc:1911:0:
../../3rdparty/chromium/third_party/skia/third_party/skcms/src/Transform_inl.h:537:6: note: 'template<int I, int B> void clut(const skcms_A2B*, I32x8, I32x8, Fx8*, Fx8*, Fx8*, Fx8)' previously declared here
 void clut(const skcms_A2B* a2b, I32 ix, I32 stride, F* r, F* g, F* b, F a) {
      ^~~~
```